File: leetcode/1. Two Sum/Brute Force/brute_force.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        for(int i=0; i<nums.size(); i++){
            for(int j=0; j<nums.size(); j++){
                if(i!=j&&target==nums[i]+nums[j]){
                    return vector<int>{i,j};
                }
            }
        }
        return vector<int>{};
    }
};

File: leetcode/1. Two Sum/Brute Force/brute_force.go
func twoSum(nums []int, target int) []int {
	for i := range nums {
		for j := 1; j < len(nums); j++ {
			if i != j && nums[i]+nums[j] == target {
				return []int{i, j}
			}
		}
	}
	return []int{}
}
